### ISSUE TITLE
Whisper window still appears despite all whispers being blocked

### ISSUE THREAD

**Issue by JeeveStobs** [funkydude/BadBoy#166]
## Describe the problem
Whisper windows still appears despite all whispers being blocked by BadBoy_Levels

### What steps will reproduce the problem?

1.  Install BadBoy and BadBoy_Levels plugin
2.  Enable option to Block All Whispers
3.  Request stranger to send unsolicited whisper to test functionality (example ask in Trade chat)

### Consider attaching a screenshot below to help describe your issue (Attach directly, do not link to other websites)
![image](https://github.com/funkydude/BadBoy/assets/15347708/ef399180-ec2c-4755-ad8b-b1f68794daaf)
![image](https://github.com/funkydude/BadBoy/assets/15347708/d3a64310-03b9-457f-8745-edd5557cfada)

### What version of the addon are you using? (Stating 'latest' is not useful)
10.2 production

### Do you have an error log of what happened?
N/A

### Any additional information? (example: WoW language if not English)
Enabled Block All Whispers and still see a whisper window (tab) notification for a whisper. The actual whisper message is not display however. See attached screenshots.

**Comment by funkydude** [comment]
Type `/dump GetAutoCompleteRealms()`
Post a screenshot of the results.
What realms were those players from, realms that were included in the list of the dump above?

**Comment by JeeveStobs** [comment]
![image](https://github.com/funkydude/BadBoy/assets/15347708/73d0d3d2-9918-498a-aa7f-66cafe0222e9)
Screenshot attached

Players were from Daggerspine and Bonechewer. My character is from Aegwynn.

**Comment by funkydude** [comment]
Did this just start happening as of 10.2?

**Comment by JeeveStobs** [comment]
I noticed this after 10.2, yes. I don't know if it started in 10.1.7 as that is when I changed the "censor messages" setting that Blizzard implemented. Besides that, no other changes to my UI settings pertaining to whispers or anything else.

**Comment by JeeveStobs** [comment]
Happening today in PvP battleground. Rogue sent a hate tell and I see and whisper window (tab) but no message.

Does the "New Whispers" setting in Options > Gameplay > Social need to be set to In-line instead of New Tab for the BadBoy addon to work correctly?

**Comment by JeeveStobs** [comment]
Appears to work correctly if "New Whispers" setting in Options > Gameplay > Social need to be set to In-line instead of New Tab.

Will update setting on my other chars.

**Comment by funkydude** [comment]
No, it is not intended that you HAVE to use that setting, that is just a useful workaround, the issue is still valid.
